src/main/java/org/zalando/nakadi/validation/JsonSchemaEnrichment.java

```diff
@@ -164,15 +171,25 @@ private static JSONObject addMetadata(final JSONObject schema, final EventType e
         metadataProperties.put("parent_eids", arrayOfUUIDs);
         metadataProperties.put("flow_id", string);
         metadataProperties.put("partition", string);
+        metadataProperties.put("partition_compaction_key", string);
```

Comment:
I guess you are putting it twice. Here and in line 181.

Reply:
🤦‍♂️ 
Fixed https://github.com/zalando/nakadi/pull/907/commits/5363d3ff43bbfeea7dad99c638ff585d1d118e9f